addons/mrp_repair/tests/test_flows.py

```diff
@@ -0,0 +1,46 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import Form, tagged
+from odoo.addons.mrp.tests.common import TestMrpCommon
+
+
+@tagged('post_install', '-at_install')
+class TestMrpRepairFlows(TestMrpCommon):
+    def test_possible_to_add_kit_after_confirm(self):
+        """
+        Test that it is possible to add a kit manufactured product to an already confirmed Repair Order
+        """
+        repaired, part1, part2, kit = self.env['product.product'].create([
+            {'name': 'Repaired'},
+            {'name': 'Kit Component1'},
+            {'name': 'Kit Component2'},
+            {'name': 'Kit', 'type': 'product'},
+        ])
+        CREATE = 0
+        self.env['mrp.bom'].create({
+            'product_id': kit.id,
+            'product_tmpl_id': kit.product_tmpl_id.id,
+            'type': 'phantom',
+            'bom_line_ids': [
+                (CREATE, 0, {
+                    'product_id': part1.id,
+                    'product_tmpl_id': part1.product_tmpl_id.id,
```

Comment:
Setting the template is useless, and imo setting the quantity would make it clearer
```suggestion
                    'product_qty': 1.0,
```